mava/configs/default/ff_masac.yaml

```diff
@@ -3,8 +3,8 @@ defaults:
   - logger: logger
   - arch: anakin
   - system: sac/ff_masac
-  - network: continuous_mlp  # [continuous_mlp]
-  - env: mabrax  # [mabrax, mpe]
+  - network: mlp
+  - env: mabrax  # [mabrax]
```

Comment:
Why remove mpe here and in ISAC?

Reply:
This was by mistake I guess when I was solving merge conflicts from develop 🤦‍♀️